### PR TITLE
Enhancement confluence openapi plugin

### PR DESCRIPTION
**Pull request by devMaFi**
related to issue #616 

* [x] Does your PR affect the documentation?
* [x] If yes, did you update the documentation or create an issue for updating it?

### Changes to Core Features:

* [x] Have you written new tests for your core changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### Change to Documentation:

* [x] Did you build the docs and copy them to the `/docs` folder?

### CODE REVIEW

**Comment by rdmueller** [comment]
Hi @devMaFi ,
thanx for this PR. Currently, the diff looks a bit overcomplicated and I've already found two sources for that:

I made the mistake that I didn't update the PR-Request template and it still states that we use the `/doc`-folder. We switched to github actions which deploy to gh-pages. Same result. Different process.

Could you please remove the `/doc` folder from your PR (just remove it from your repository branch it and will also be removed from this PR).

Next thing I saw is that you checked in your eclipse config files. I think we don't neet `.settings`, `.project`, `.classpath` and `.DS_Store`. Could you be so kind and remove them too?

And I hope you added your name to the list of contributors: https://github.com/docToolchain/docToolchain/blob/master/src/docs/manual/05_contributors.adoc :-)

Ralf

**Comment by devMaFi** [comment]
Hi @rdmueller 

thanks for sharing the feedback. I have deleted the /docs folder and the mentioned files from created by my IDE.

And sure, I have added myself to the list.

Martin

**Comment by rdmueller** [comment]
thanx. I now checked the other files and removed some test files which I think shouldn't be part of the PR. To be honest, I don't know how they got there...

There are still some `.DS_Store` files which I can't easily remove...

**Comment by rdmueller** [comment]
Ha! Managed to fix it through gitpod... 
next step: check code...

**Comment by devMaFi** [comment]
@rdmueller Thanks for taking care and deleting the files!
In case of questions, I'm happy to help!

File: src/docs/manual/03_task_publishToConfluence.adoc [hunk 1]
```diff
@@ -88,17 +88,45 @@ If you need to provide a proxy to access Confluence, you may set a map with keys
 
 useOpenapiMacro::
 If this option is present and equal to "confluence-open-api" then any source block marked with class openapi will be wrapped in Elitesoft Swagger Editor macro:  (see https://marketplace.atlassian.com/apps/1218914/open-api-swagger-editor-for-confluence[Elitesoft Swagger Editor]) +
-For backward compatibility: If this option is present and equal to `true`, then again the Elitesoft Swagger Editor macro will be used. +
-If this option is present and equal to "open-api" then any source block marked with class openapi will be wrapped in Open API Documentation for Confluence macro:  (see https://marketplace.atlassian.com/apps/1215176/open-api-documentation-for-confluence[Open API Documentation for Confluence]). A download source (yaml) button is shown by default. +
-This is how you'd include your openapi YAML file: +
-+
+For backward compatibility: If this option is present and equal to `true`, then again the Elitesoft Swagger +
```
Comment (rdmueller) [inline src/docs/manual/03_task_publishToConfluence.adoc:91]: I get the feeling that there is something wrong with the documentation at this point. Can you please double-check?

**Review by rdmueller — CHANGES_REQUESTED** [review]
The changes in the source look fine to me, but I get the feeling that there is some copy&paste problem with the docs. Can you please double check this?
The Pull request will then be ready to merge.

File: src/docs/manual/03_task_publishToConfluence.adoc (hunk 1 above)
Comment (devMaFi) [reply src/docs/manual/03_task_publishToConfluence.adoc:91]: I changed the documentation for this in order to explain how to use the configuration for the OpenAPI Documentation Plugin as this differs from Confluent OpenAPI Plugin.

Bildschirmfoto 2021-09-24 um 14 08 03

I can revoke the changes, but I thought it's worth to explain how the plugin can be used.

Comment (rdmueller) [reply src/docs/manual/03_task_publishToConfluence.adoc:91]: ok

**Comment by rdmueller** [comment]
thanx. I merged your PR. I will publish a new release soon.

**Pull request merged**